src/index.ts

```diff
@@ -30,6 +30,16 @@ function emitDom() {
     const inputFolder = path.join(__SOURCE_DIRECTORY__, "../", "inputfiles");
     const outputFolder = path.join(__SOURCE_DIRECTORY__, "../", "generated");
 
+    // ${name} will be substituted with the name of an interface
+    const removeVerboseIntroductions: [RegExp, string][] = [
+        [/^(The|A) ${name} interface of (the\s*)*([a-z\s]+ API)(\\\'s)?/, 'An interface of the $3 '],
+        [/^(The|A) ${name} (interface|event|object) (is|represents|represent|describes|defines)?/, ''],
+        [/^An object implementing the ${name} interface (is|represents|represent|describes|defines)/, ''],
+        [/^The ${name} is an interface representing/, ''],
+        [/^This type (is|represents|represent|describes|defines)?/, ''],
```

Comment:
Looks like `represent` is not needed anymore for the latest MDN descriptions.

Reply:
@mihailik We still have one `represent`, is it still needed?

Reply:
I would keep it in this code. This can slip in at any time again in MDN.

Reply:
We can fix typos on MDN in any time. Better to fix on MDN rather than to workaround on TSJS.